app.py

```diff
@@ -187,12 +212,12 @@ def submit_version(self, path, path_to_upload, fields, first_frame, last_frame,
 
         return sg_version
 
-    def render_and_submit_version(self, path, fields, first_frame, last_frame, sg_publishes, sg_task,
-                                  comment, thumbnail_path, progress_cb, color_space=None, *args, **kwargs):
+    def render_and_submit_path(self, path_to_frames, fields, first_frame, last_frame, sg_publishes, sg_task, comment,
```

Comment:
Just for organization sake, please move this so its just below `render_and_submit_version()`

Reply:
Haha ok :P